source/agora/node/Registry.d

```diff
@@ -914,6 +917,9 @@ private struct ZoneData
         this.query_records_get_typed = format("SELECT address, type, ttl " ~
             "FROM registry_%s_addresses WHERE pubkey = ? AND type = ?", zone_name);
```

Comment:
Although not changed in this PR. It looks like zone_name will be used as pubkey.

Reply:
No, it is for `registry_%s_addresses`

Reply:
🤦